common/src/Model/GameFactory.cpp

```diff
@@ -241,7 +242,8 @@ namespace TrenchBroom {
                     gameConfig.setCompilationConfig(parser.parse());
                 }
             } catch (const Exception& e) {
-                throw FileDeletingException("Could not load compilation configuration '" + path.asString() + "': " + std::string(e.what()), m_configFS->makeAbsolute(path));
+                std::cerr << "Could not load compilation configuration '" + path.asString() + "': " + std::string(e.what()) << "\n";
+                gameConfig.setCompilationConfigParseFailed(true);
```

Comment:
I think that the "parse failed" flag should maybe live in `GameFactory`, e.g. in a map. Then it would not have to be a mutable member. And it's only used in `GameFactory`, so I think it belongs here more than in `GameConfig`. What do you think?

Reply:
I'm using gameConfig.setCompilationConfig() as a cache of what was last seen on disk, the "parse failed" flag should be bundled with that. 
In the Preferences code, I used a std::variant (or kdl::result, I forget) for the same purpose. I could switch to that method, but it should stay in GameConfig as long as the cached config stays there

Reply:
Ok!